independent-publisher-connectors/Mensagia/Readme.md

```diff
@@ -0,0 +1,37 @@
+# Mensagia
+Mensagia offer reliable SMS, Email and Voice for business automation communication.
+
+## Publisher: Sinergyne Global Communications, S.L.
```

Comment:
@AfriSR Please provide your name as a publisher as you are publishing it as an independent publisher connector.